go/proxyd/config.go

```diff
@@ -74,6 +74,21 @@ type Config struct {
 	WSMethodWhitelist []string            `toml:"ws_method_whitelist"`
 }
 
+type DaisyChainBackends struct {
+	Epoch1RPCURL string `toml:"epoch_1_rpc_url"`
```

Comment:
Wouldn't it be a bit cleaner to make this an array? Or do you prefer the explicitness of having a list of named epochs?

Reply:
I didn't want to make it an array so that its very explicit, since this is very consensus critical configuration from the PoV of an app dev. If there is strong preference to make it into an array, I can, but I'd like to strategically minimize the amount of user config error if possible

Reply:
I've removed the concept of a different config struct, the same config is now used